go.mod

```diff
@@ -12,6 +13,7 @@ require (
 	github.com/onsi/gomega v1.10.2
 	github.com/prometheus/common v0.13.0
 	github.com/smartystreets/goconvey v1.6.4
+	github.com/versent/unicreds v1.5.0
```

Comment:
I saw there's a version [1.7.0](https://github.com/Versent/unicreds/releases/tag/1.7.0) for this library, maybe we could use it?

Reply:
Will do!

Reply:
So, I try to use 1.7.0, but go mod keeps changing it to v1.5.1-0.20180327234242-7135c859e003 . I think the maintainers changed the way they name the releases and go mod only likes versions starting with a "v".

![image](https://user-images.githubusercontent.com/2432275/102111309-82e2f480-3e36-11eb-9322-3c08bf1897cc.png)

Reply:
Agreed. There's even an [issue](https://github.com/Versent/unicreds/issues/84) for that.

One way one can do is to use the commit hash, like `go get github.com/versent/unicreds@7135c85`, this will create a pseudo-version name like `v1.5.1-0.20180327234242-7135c859e003` (the pseudo-version name gets some more chars from the hash string). This is documented [here](https://golang.org/ref/mod#versions), if you wanna give it a try.